.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: http://EditorConfig.org
+# Since OBS follow the Linux Kernel Conventions
```

Comment:
Spelling/Grammar: "follow" should be "follows"

These lines (1-4, or 2-4) could be wrapped much wider, as they have about 30 characters to spare before hitting the 80 character limit.

Also, I'm not sure about using "Linux Kernel Conventions"?  As far as I know, the phrase describing the coding style of OBS is one of these:

- kernel normal form (linux variant)
- KNF (Linux)
- Linux kernel coding style

See [CONTRIBUTING.rst](https://github.com/jp9000/obs-studio/blob/master/CONTRIBUTING.rst#coding-guidelines).